### PR TITLE
Add of cycle counter

### LINKED ISSUE #4: [FEATURE] Cycle counter
## 🚀 Feature

A banner should display a counter with the number of cycles the user did. 

### Motivation

Let the user track his progress.

### PR DESCRIPTION
**Pull request by JM-Pascual**
Add of the feature that allows for the timer to keep track of cycles.

## Motivation

Closes #4 

## Summary of changes

Now the timer keeps track of how many pomodoros where performed.

### CODE REVIEW

File: src/components/timer.js [hunk 1]
```diff
@@ -63,14 +65,15 @@ class Timer extends Component {
...
-    const { mode, time } = this.state;
+    const { mode, time, pomodoroCount } = this.state;
 
     return (
       <div className="timer-container">
         <div className="timer-mode">{mode}</div>
         <div className="timer-time">
           {Math.floor(time / 60)}:{(time % 60).toLocaleString('en-US', { minimumIntegerDigits: 2 })}
         </div>
+        <div className="pomodoro-counter">Pomodoros completed: {pomodoroCount}</div>
```
Comment (joac001) [inline src/components/timer.js:76]: should be 'completed pomodoros' instead of 'Pomodoros completed'

Comment (joac001) [reply src/components/timer.js:76]: before i can approve the pr merge this branch with main.

Comment (JM-Pascual) [reply src/components/timer.js]: Changes are now pushed! Waiting 4 a review on them 👍🏼

**Review by joac001 — APPROVED** [review]
Everything looks fine. Last reviews were solved. Ready to merge.

**Pull request merged**